src/dashboard/src/components/api/views.py

```diff
@@ -655,33 +660,33 @@ def path_metadata(request):
     path = request.GET.get('path', '') if request.method == 'GET' else request.POST.get('path', '')
 
     # Get current metadata, if any
-    file_lod = get_object_or_None(models.SIPArrange, arrange_path=path, sip_created=False)
-    if file_lod is None:
-        # Try with trailing / to see if it's a directory
-        file_lod = get_object_or_None(models.SIPArrange, arrange_path=path + '/', sip_created=False)
+    files = models.SIPArrange.objects.filter(
+        arrange_path__in=(path, path + '/'),
+        sip_created=False)
+    if not files:
+        raise django.http.Http404
+    file_lod = files.first()
```

Comment:
So I don't know if this is ever going to happen but I think this pretty much mimics the behavior found before this change.

Reply:
Right... previously attribute access on `None` would have triggered an exception and Django probably would have returned a 500 response (?)